superbit_lensing/shear_profiles/annular_jmac.py

```diff
@@ -14,7 +14,7 @@
 import pudb, pdb
 from esutil import htm
 
-from shear_plots import ShearProfilePlotter
+# from superbit_lensing.shear_profiles.shear_plots import ShearProfilePlotter
```

Comment:
I don't understand why we no longer need the plotter class...

Reply:
This was a mistake! Thank you for catching this.